pkg/operator/sync.go

```diff
@@ -281,11 +285,57 @@ func (optr *Operator) syncRenderConfig(_ *renderConfig) error {
 		templatectrl.BaremetalRuntimeCfgKey:      imgs.BaremetalRuntimeCfg,
 	}
 
+	ignitionHost, err := getIgnitionHost(&infra.Status)
+	if err != nil {
+		return err
+	}
+
+	masterPointerConfig, err := ctrlcommon.PointerConfig(ignitionHost, "master", rootCA)
+	if err != nil {
+		return err
+	}
+	masterPCData, err := json.Marshal(masterPointerConfig)
+	if err != nil {
+		return err
+	}
+	workerPointerConfig, err := ctrlcommon.PointerConfig(ignitionHost, "worker", rootCA)
+	if err != nil {
+		return err
+	}
+	workerPCData, err := json.Marshal(workerPointerConfig)
+	if err != nil {
+		return err
+	}
+
 	// create renderConfig
-	optr.renderConfig = getRenderConfig(optr.namespace, string(kubeAPIServerServingCABytes), spec, &imgs.RenderConfigImages, infra.Status.APIServerInternalURL)
+	optr.renderConfig = getRenderConfig(optr.namespace, string(kubeAPIServerServingCABytes), spec, &imgs.RenderConfigImages, infra.Status.APIServerInternalURL, masterPCData, workerPCData)
 	return nil
 }
 
+func getIgnitionHost(infraStatus *configv1.InfrastructureStatus) (string, error) {
+	internalURL := infraStatus.APIServerInternalURL
+	internalURLParsed, err := url.Parse(internalURL)
+	if err != nil {
+		return "", err
+	}
+	ignitionHost := fmt.Sprintf("%s:22623", internalURLParsed.Hostname())
+	switch infraStatus.PlatformStatus.Type {
+	case configv1.BareMetalPlatformType:
+		// Baremetal needs to point directly at the VIP because we don't have a
+		// way to configure DNS before Ignition runs.
```

Comment:
(Actually they do since https://github.com/openshift/enhancements/pull/210 )

Reply:
it was merely copied over from the installer, will drop!